src/server/routes/store.js

```diff
@@ -1,13 +0,0 @@
-import { Router } from 'express';
-import { json } from 'body-parser';
-
-import {
-  signAgreement
-} from '../handlers/store';
-
-const router = Router();
-
-router.use('/store/agreement', json());
-router.post('/store/agreement', signAgreement);
-
-export default router;
```

Comment:
If router is now fully removed shouldn't `handlers/store.js` be removed as well?